src/DssSpell.sol

```diff
@@ -106,226 +104,303 @@ contract DssSpellAction is DssAction {
-            input[0] = PoolConfiguratorLike.InitReserveInput({
-                aTokenImpl: SPARK_ATOKEN_IMPL,
-                stableDebtTokenImpl: SPARK_STABLE_DEBT_TOKEN_IMPL,
-                variableDebtTokenImpl: SPARK_VARIABLE_DEBT_TOKEN_IMPL,
-                underlyingAssetDecimals: GemAbstract(token).decimals(),
-                interestRateStrategyAddress: SPARK_INTEREST_RATE_STRATEGY,      // Dummy strategy - compare to other borrow-disabled asset like sDAI
-                underlyingAsset: token,
-                treasury: SPARK_TREASURY,
-                incentivesController: address(0),
-                aTokenName: "Spark GNO",
-                aTokenSymbol: "spGNO",
-                variableDebtTokenName: "Spark Variable Debt GNO",
-                variableDebtTokenSymbol: "variableDebtGNO",
-                stableDebtTokenName: "Spark Stable Debt GNO",
-                stableDebtTokenSymbol: "stableDebtGNO",
-                params: ""
-            });
-            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).initReserves(input);
-            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).configureReserveAsCollateral({
-                asset: token, 
-                ltv: 20_00,
-                liquidationThreshold: 25_00,
-                liquidationBonus: 110_00
-            });
-            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).setDebtCeiling(token, 5 * MILLION * HUNDRED);
-
-            address[] memory tokens = new address[](1);
-            tokens[0] = token;
-            address[] memory oracles = new address[](1);
-            oracles[0] = SPARK_GNO_ORACLE;
-            AaveOracleLike(SPARK_AAVE_ORACLE).setAssetSources(
-                tokens,
-                oracles
+        // --- BlockTower Vault Debt Ceiling Adjustments ---
+        // Poll: https://vote.makerdao.com/polling/QmPMrvfV#poll-detail
+        // Forum: https://forum.makerdao.com/t/blocktower-credit-rwa-vaults-parameters-shift/20707
+
+        (uint256 RWA010_A_ART, , , ,) = VatLike(MCD_VAT).ilks("RWA010-A");
+        (uint256 RWA011_A_ART, , , ,) = VatLike(MCD_VAT).ilks("RWA011-A");
+
+        if (RWA010_A_ART + RWA011_A_ART == 0) {
+            // Decrease the Debt Ceiling (line) of BlockTower S1 (RWA010-A) from 20 million DAI to 0 DAI.
+            DssExecLib.setIlkDebtCeiling("RWA010-A", 0);
+            // Decrease the Debt Ceiling (line) of BlockTower S2 (RWA011-A) from 30 million DAI to 0 DAI.
+            DssExecLib.setIlkDebtCeiling("RWA011-A", 0);
+            // Increase the Debt Ceiling (line) of BlockTower S3 (RWA012-A) from 30 million DAI to 80 million DAI.
+            // Note: Do not increase global Line because there is no net change from these operations
+            DssExecLib.setIlkDebtCeiling("RWA012-A", 80 * MILLION);
+
+            // Increase the price to enable DAI to be drawn -- value corresponds to
+            // [ (Debt ceiling) + (RWA stability fee ) ^ (minimum deal duration in years) ] * liquidation ratio
```

Comment:
My apologies here, I had made a typo in the formula.
The correct formula is

`Debt ceiling * [ (1 + RWA stability fee ) ^ (minimum deal duration in years) ] * liquidation ratio`

Sorry for this